spec/models/like_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'Validations' do
+    let(:user) do
+      User.create(name: 'Rishi', photo: 'Image will be displayed here', bio: 'Hello guys', posts_counter: 0)
```

Comment:
- [ ] Here Also, Please add a test for the `update_like_count` method inside your `Like` model. We need to make sure all custom methods are tested.